website_snippet_product_category/static/src/scss/snippet.scss

```diff
@@ -0,0 +1,37 @@
+.categ_container {
+    .categ_scroll_wrapper {
+        border: 0 solid $border-color;
+        border-width: 0 1px 0 0;
+    }
+    .categ_scroll {
+        overflow: hidden;
+        overflow-y: auto;
+    }
+
+    .categ_img img {
+        width: 1.4em;
+    }
+    .categ_tree_level {
+        &[data-tree-level="2"] {
+            font-size: 0.9em;
+            .categ_link {
+                text-transform: capitalize;
+                color: $gray-800;
```

Comment:
Have you considered to use [BS text classes](https://getbootstrap.com/docs/4.5/utilities/colors/#color) instead? I think that being standard BS4 classes will make it easier that their colors fit with any BS theme out there.